README.md

```
@@ -1,34 +1,64 @@
+mise exec pnpm dev
+
+# Build for production
+mise exec pnpm build
+
+# Type check
+mise exec pnpm check
+```
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)